packages/discord-verify/src/node.ts

```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+// @ts-ignore-error Node Crypto types are not well defined yet
+import crypto from "node:crypto";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+// @ts-ignore-error Node Process
```

Comment:
Why?

Reply:
I think I can remove this after we merge the other PR (bumps dependencies)

Reply:
Mean the second one

Reply:
Oh and I guess we can't use types/node anyways. Yeah fixing this